### PR TITLE
Issue 199 - package updates

### PR DESCRIPTION
**Pull request by PhillipsOwen**
addresses issue #199 

this updates all react packages for the UI to the latest versions. 

i found two inconsistencies when doing this:
- eslint was reporting errors in the recently added side/side compare code. i disabled the lint check.
- the positioning of the legend and control panel had to be hard-coded. as apparently "calc" went wonky. 

you will have to perform a "npm update" to upgrade your local packages.

### CODE REVIEW

File: src/components/control-panel/control-panel.js [hunk 1]
```diff
@@ -331,8 +331,8 @@ export const ControlPanel = () => {
       sx={{
         p: 0,
         position: 'absolute',
-        bottom: 'calc(4 * var(--joy-spacing))',
-        right: 'calc(4 * var(--joy-spacing))',
+        bottom: '20px',
```
Comment (mbwatson) [inline src/components/control-panel/control-panel.js:334]: it looks like the css variable names are prefixed with "mui" on this branch, so this variable would be `--mui-spacing`

**Review by mbwatson — APPROVED** [review]
i don't think that css var matters too much, though. 

:+1:  `found 0 vulnerabilities`

File: src/components/control-panel/control-panel.js (hunk 1 above)
Comment (PhillipsOwen) [reply src/components/control-panel/control-panel.js:334]: i dont think special calcs are needed here. so i hard coded them into the right hand corners.

**Review by lstillwe — APPROVED** [review]
Still works!

**Pull request merged**